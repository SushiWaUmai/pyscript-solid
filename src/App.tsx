import PyEnv from "./components/PyEnv";
import PyScript from "./components/PyScript";

function App() {
  return (
    <div>
      <PyEnv>- numpy</PyEnv>
      <PyEnv>- pandas</PyEnv>
      <PyScript>print("Hello World")</PyScript>
    </div>
  );
}

export default App;
