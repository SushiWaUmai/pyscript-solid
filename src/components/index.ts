export * from "./PyBox";
export * from "./PyButton";
export * from "./PyConfig";
export * from "./PyEnv";
export * from "./PyInputBox";
export * from "./PyRegisterWidget";
export * from "./PyRepl";
export * from "./PyScript";
export * from "./PyTitle";
