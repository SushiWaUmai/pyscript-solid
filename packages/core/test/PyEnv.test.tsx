import { describe, expect, test } from "vitest";
import { render } from "solid-testing-library";
import { PyEnv } from "../src";

describe("<PyEnv />", () => {
  test("renders", () => {
    const { unmount, container } = render(() => <PyEnv>- numpy</PyEnv>);
    expect(container.innerHTML).toMatchSnapshot();
    unmount();
  });
});
