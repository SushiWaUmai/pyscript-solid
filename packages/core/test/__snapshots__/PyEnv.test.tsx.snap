// Vitest Snapshot v1

exports[`<PyEnv /> > renders 1`] = `"<py-env>- numpy</py-env>"`;
